integration/spark/src/test/common/java/io/openlineage/spark/agent/lifecycle/SparkReadWriteIntegTest.java

```diff
@@ -53,18 +61,24 @@
 import org.apache.spark.sql.types.StructType;
 import org.codehaus.jackson.JsonGenerator;
 import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.io.TempDir;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.DockerImageName;
 import scala.Tuple2;
 import scala.collection.immutable.HashMap;
 
 @ExtendWith(SparkAgentTestExtension.class)
 public class SparkReadWriteIntegTest {
 
+  private final KafkaContainer kafkaContainer =
+      new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:7.0.0"));
```

Comment:
Kafka version here is different than in `SparkContainerIntegrationTest`. Is there a reason for it?

Reply:
I was debugging something that wasn't working and tried upgrading the Kafka version. It was completely unrelated, but TBH, I don't know why the version in the `SparkContainerIntegrationTest` is what it is. I'll try upgrading it to the latest version and see if the tests still pass.